### ISSUE TITLE
Weakaraus wiped, can't create new ones (classic)

### ISSUE THREAD

**Issue by powntus1838** [WeakAuras/WeakAuras2#2545]
I was in a BG and got a window popup saying I experienced lots of LUA errors but ignored it for the time being. Shortly after all my WA's got wiped and I can't create new ones, none of the buttons in game work and I can't import either. Absolutely nothing happens. I've tried reinstalling the addon, I deleted my saved variables, everything but nothing seem to work.

**Comment by powntus1838** [comment]
Seems to be an issue when you use something that skins the WA icons. That's what I've gathered so far atleast.

**Comment by mrbuds** [comment]
1. We need an error log
Please install https://www.curseforge.com/wow/addons/bugsack & https://www.curseforge.com/wow/addons/bug-grabber
Note that if the error looks like `[string "--[[ Error in ' my awesome aura' ]` then the bug is in the aura that got mentioned, not in WeakAuras itself

2. Try with only weakauras addon enable, and update all your addons, primarily Masque

**Comment by powntus1838** [comment]
Okey, I'm sorry then because I don't think I can provide that. After hours of trial and error I've finally managed it to work again and working on remaking all my WA's currently and I don't want to mess it up again.

**Comment by mrbuds** [comment]
I highly recommend to keep backup of your auras.
WeakAuras Companion can handle it for you for wa and plater, you can get it at https://weakauras.wtf/

**Comment by powntus1838** [comment]
So it happened again and this time I installed bugsack and buggrabber and this is the error i'm getting https://pastebin.com/7dvibkjU

**Comment by InfusOnWoW** [comment]
Is taht the first error in the bugsack?

**Comment by powntus1838** [comment]
Yes, it is.

**Comment by powntus1838** [comment]
Yes, it is. First and only

**Comment by powntus1838** [comment]
But for some reason I can bypass the error. If I disable all addons except for Weakaruas upon logging in, it works fine. And then just enable all the addons that I need. If, however, I log in with everything enabled from the start, I get this error.

**Comment by Stanzilla** [comment]
Which other addons are you testing with?

**Comment by powntus1838** [comment]
I've tested them all one by one to see if the error triggers but it doesn't, only seem to be when I first log in whith everything enabled.
- Elvui
- DBM
- EXRT
-AddonSkins
- SharedMedia
- WeaponSwingTiemr
- TC2
- MSBT
- Nova World buff
- Leatrix Plus and Maps
- Details 
- Atlasloot
- Auctionator

**Comment by Stanzilla** [comment]
Try without addon skins

**Comment by powntus1838** [comment]
I have tried, same thing.

**Comment by powntus1838** [comment]
I can however provide with more error now cause I just got a few others when I logged in this morning. I'll post them in the order they came from BugSack.
1. https://pastebin.com/Svk37uqT
2. https://pastebin.com/U5NG4iHq
3. https://pastebin.com/YPhXWJki
4. https://pastebin.com/qPE6m3yJ
5. https://pastebin.com/LhduYnhs
6. https://pastebin.com/v79chHwh
7. https://pastebin.com/eeN05DWL
8. The very first one I posted that keeps reaccuring every 3-4 seconds.

**Comment by InfusOnWoW** [comment]
Everything in those errors point to a very broken WeakAuras isntallation. Or since it it only happebs with other addons enabled with a severe bug in one of those addons.

**Comment by powntus1838** [comment]
I did a clean re-install of the addon after the update (since I at first thought everything was gone already), Deleted my Savied Variables and all that jazz. But I'm assuming it doesn't to exactly which addon?

**Comment by powntus1838** [comment]
I think I might have located the problem. I had an addon enabled (that I used many many months ago) but actually never disabled. SimpleUnitFrames. I found it now when I was enabling them one by one (in the character selection screen) to see if I would get an error. And then I enabled it and logged in I got the error, logged out and disabled, and didn't get an error when I logged back on. Seems it is very outdated aswell and I don't really use it either so I'll just delete it completely.
